src/STRPAK.CPP

```diff
@@ -1332,6 +1332,22 @@ BOOL strMatch(					// string match
 			bSeenDifSpace = TRUE;		// both space but different
 	}
 }			// strMatch
+//-----------------------------------------------------------------------------
+int strcmpi(		// Substitude windows strcmpi functions
```

Comment:
I believe that what Chip was suggesting was to make this function do two different things, but at compile-time. If we're on a Windows system, this function calls the existing `_stricmp`. If not, it uses the new function.

Reply:
This is not quite what I was proposing.

Proposal #1 -- use the Microsoft name and implement our own if needed.  That is

#if !defined( _WIN32)
int stricmp(  whatever)
our own code
#endif

The idea is to avoid the forwarded call when on Windows (in all cases, in fact).

Proposal #2 -- map the Microsoft name to another implementation if needed

#if !defined( _WIN32)
#define _stricmp( s1, s2)  strcasecmp( s1, s2)   // use POSIX implementation
#endif

Or we change all the calls to strcasecmp() and do the reverse definition -- see https://stackoverflow.com/questions/3694723/error-c3861-strcasecmp-identifier-not-found-in-visual-studio-2008
#ifdef _MSC_VER 
//not #if defined(_WIN32) || defined(_WIN64) because we have strncasecmp in mingw
#define strncasecmp _strnicmp
#define strcasecmp _stricmp
#endif

The ideas are 1) don't make chained calls and 2) attempt to avoid implementing our own.

Reply:
I'm of two minds about this. It's fine to try and eliminate chained calls, but I can't imagine the compiler isn't optimizing that out? I find the price you pay in navigation and confusion when you realize that a legitimate function name is not, in fact, calling the function you thought it was, is generally too high for my liking.

Reply:
For now, I will implement option 1 because it will be challenging to test option 2 since I would like to at least build the program in MAC or Linux.  I will leave a comment to implement option 2 if we decide to go that route.